### DISCUSSION TITLE
Simpler way to get all the field names

### DISCUSSION THREAD

**Discussion by github-0609** [pydantic/pydantic#7036] [Question] [answered]
variables = []
for key in obj.model_fields.keys():
        variables.append(key)
for key in obj.__pydantic_extra__.keys():
        variables.append(key)
        
I must be missing a easier way for this

**Comment by lmmx** [comment, marked as answer]
Iterating over dicts gives keys so `[*obj.model_fields]` or `list(obj.model_fields)`
